partials/protocols.html

```diff
@@ -0,0 +1,81 @@
+{% macro displayProtocol(protocol, protocolVersion) %}
+    {% if protocolVersion %}
+    <span class="bg-teal-500 font-bold no-underline text-white uppercase rounded ml-2"
+          style="height: 20px;font-size: 11px;padding: 3px;">{{ protocol }}
+        {{ protocolVersion }}</span>
+    {% else %}
+    <span class="bg-teal-500 font-sans font-bold no-underline text-white uppercase rounded ml-2"
+          style="height: 20px;font-size: 11px;padding: 3px;">{{ protocol }}</span>
+    {% endif %}
+{% endmacro %}
+
+
+{% macro bindings(eltName, elt, odd=false) %}
```

Comment:
what elt stands for?

Reply:
like `element`? if they then better use `prop` that is common in other macros in this template

Reply:
yes this is for `element` because this macro can take a `Message` element, or a `Channel` or an `Operation` ... etc
are you sure that `prop` fit to this use case ?

Reply:
done